pkg/skaffold/build/local/jib_gradle_test.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package local
+
+import (
+	"context"
+	"io/ioutil"
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/docker"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+	"github.com/pkg/errors"
+)
+
+func TestBuildJibGradle(t *testing.T) {
+	var tests = []struct {
+		description   string
+		artifact      *latest.JibGradleArtifact
+		cmd           util.Command
+		shouldErr     bool
+		expectedError string
+	}{
+		{
+			description: "local build",
+			artifact:    &latest.JibGradleArtifact{},
+			cmd:         testutil.FakeRun(t, "gradle -Djib.console=plain :jibDockerBuild --image=img:tag"),
```

Comment:
I'm not sure how this testing framework works: Are we verifying that the code creates the command defined in FakeRun `"gradle -Djib.console=plain :jibDockerBuild --image=img:tag"`?

Reply:
It’s checking that command and only this one runs

Reply:
kinda like a `Mockito.when().then()`?

Reply:
Yes.